doc/releases/release-notes-3.1.rst

```diff
@@ -459,6 +465,13 @@ Drivers and Sensors
 
 * GPIO
 
+  * Refactored GPIO flags. Upper 8 bits of ``gpio_dt_flags_t`` is now reserved
+    for controller/SoC specific flags and certain hardware-specific flags
+    defined as common so far (IO voltage level, drive strength, debounce filter)
+    were replaced with ones defined in this controller/SoC specific space.
+  * Added Xilinx PS MIO/EMIO GPIO controller driver.
```

Comment:
I'm pretty sure this is just MIO, not EMIO.

Reply:
I took this information from https://github.com/zephyrproject-rtos/zephyr/commit/ca3390524815520c78e7bdc60990385e9a332ad2. It mentions both MIO and EMIO everywhere.

Reply:
Got it.